lib/class-wp-json-authentication-oauth1.php

```diff
@@ -547,11 +547,19 @@ protected function check_oauth_signature( $consumer, $oauth_params, $token = nul
 			case 'PUT':
 				$params = wp_unslash( $_POST );
 				break;
+			default:
+				return new WP_Error( 'json_oauth1_unknown_http_method', __( "Unknown http method: $http_method" ),
```

Comment:
This needs to use `sprintf( __( 'Unknown HTTP method: %s' ), $http_method )` to ensure it can be translated correctly. Additionally, if you're splitting this over multiple lines, please split all params out onto separate lines:
```php
return new WP_Error(
	'json...',
	sprintf( __( '...' ) ),
	array( ... )
);
```